admin/tools/docker/replication/deployment/ncsa/start.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This file is part of {{ cookiecutter.package_name }}.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+# Load parameters of the setup into the corresponding environment
+# variables
+
+. $(dirname $0)/env_svc.sh
+
+# Start database services on the master node
+#
+# The general log is temporarily disabled because it results in a huge size
+# of the log file.
+#
+#   --general-log --general-log-file="${DB_DATA_DIR}/log/${DB_CONTAINER_NAME}.general.log" \
+
+if [ ! -z "${DB_SERVICE}" ]; then
```

Comment:
`-n` is a shorter variant of `! -z`

Reply:
Agreed. Replaced in this and other locations.